compiler-rt/lib/sanitizer_common/sanitizer_symbolizer_markup.cpp

```diff
@@ -119,11 +119,10 @@ static void RenderMmaps(InternalScopedString *buffer,
     // module.base_address == dlpi_addr
     // range.beg == dlpi_addr + p_vaddr
     // relative address == p_vaddr == range.beg - module.base_address
-    buffer->AppendF(
-        kFormatMmap, reinterpret_cast<void *>(range.beg),
-        static_cast<unsigned int>(range.end - range.beg),
-        static_cast<int>(moduleId), accessBuffer.data(),
-        static_cast<unsigned int>(range.beg - module.base_address()));
+    buffer->AppendF(kFormatMmap, reinterpret_cast<void *>(range.beg),
+                    static_cast<size_t>(range.end - range.beg),
```

Comment:
no need to cast size_t

I am saying that compiler-rt (sanitizers) use uptr to format as size_t.
It's all over the code, it should be OK to pass uptr into %zx.